example/app.js

```diff
@@ -28,7 +28,6 @@ const instructions = Platform.select({
 export default class App extends Component<{}> {
   render() {
     Carnival.setDisplayInAppNotifications(true);
```

Comment:
Does `Carnival.setDisplayInAppNotifications(true);` still working being called "after" `startEngine`?

Reply:
Good catch, true will work as it's the default setting but false won't. I'll remove the RN method so it has to be called using the native method.